pkg/ddevapp/testdata/TestApptypeDetection/drupal6/misc/ahah.js

```diff
@@ -234,7 +234,7 @@ Drupal.ahah.prototype.success = function (response, status) {
  */
 Drupal.ahah.prototype.error = function (response, uri) {
   alert(Drupal.ahahError(response, uri));
-  // Resore the previous action and target to the form.
+  // Restore the previous action and target to the form.
```

Comment:
This is actual Drupal 6 code, no need to be correcting it here. Not that it matters.
```suggestion
// Resore the previous action and target to the form.
```